### ISSUE TITLE
Acc & checkpoint during training

### ISSUE THREAD

**Issue by DongWooLee-Eli** [ryoungj/BoLT#2]
Hello,

I run training script by 'bash bin/run_train_bootstrap.sh run bootstrap_fixed_data.sh bootstrap_latent 4' but logged acc were always 0 and no ckpt were saved ('~/consolidated.pth' are needed to run evaluation) so 'bash bin/run_finetune_eval.sh run math bootstrap_fixed_data.sh bootstrap_latent 4' fails. How can I fix it?

Thanks in advance

**Comment by ryoungj** [comment]
If you have not seen issues in previous iterations or during warmup, the problem probably came from the particular training run as the only thing that changed is the training data. Since no checkpoints were saved, I would check the logs of the training & evaluation jobs for potential issues, such as running out of disk space or failure to launch eval jobs.
